CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Welcome to Kotary docs contributing guide
+
+Thank you for investing your time in contributing to our project!
+
+In this guide you will get an overview of the contribution workflow from opening an issue, creating a PR, reviewing, and merging the PR.
+
+
+## New contributor guide
+
+To get an overview of the project, read the [README](README.md) file. Here are some resources to help you get started with open source contributions:
+
+### Issues
+
+#### Create a new issue
+
+If you spot a problem with the docs or if a related issue doesn't exist, you can open a new issue. 
+
+#### Solve an issue
+
+Scan through our existing issues to find one that interests you. As a general rule, we don’t assign issues to anyone. If you find an issue to work on, you are welcome to open a PR with a fix.
```

Comment:
"If you find an issue to work on, you are welcome to open a PR with a fix."

You can remove the first sentence, and on top of it, there is a typo on the " ' " sign. (Incorrect character)